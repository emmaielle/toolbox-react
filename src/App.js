import React, { Component } from 'react';
import { Route, Switch } from 'react-router-dom';
import HomeComponent from './containers/home';
import './styles/App.css';

class App extends Component {
  render() {
    return (
      <Switch>
        <Route
            path="/"
            component={HomeComponent}
        />
      </Switch>
    );
  }
}

export default App;
