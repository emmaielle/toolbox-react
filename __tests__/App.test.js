// MyComponent.test.js
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from '../src/App';

describe("App component", () => {
  it("should render the App component", () => {
    const wrapper = shallow(<App />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();

  });
});