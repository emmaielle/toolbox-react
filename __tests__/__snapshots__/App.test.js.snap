// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render the App component 1`] = `
<Switch>
  <Route
    component={[Function]}
    path="/"
  />
</Switch>
`;
